File: tasks/prac2/task9.py
number = float(input("Введите число с плавающей точкой: "))

rounded_number = round(number)

result = number + rounded_number
print(f"Исходное число: {number}")
print(f"Округленное число: {rounded_number}")
print(f"Сумма: {result}")

File: tasks/prac2_js/task9.js
const prompt = require('prompt-sync')();

let number = parseFloat(prompt("Введите число с плавающей точкой: "));
let rounded = Math.round(number);

let result_sum = number + rounded;
console.log(`Исходное число: ${number}`);
console.log(`Округленное число: ${rounded}`);
console.log(`Сумма: ${number} + ${rounded} = ${result_sum}`);